pkg/controller/controller.go

```diff
@@ -0,0 +1,43 @@
+package controller
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/water-hole/ansible-operator/pkg/runner"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	crthandler "sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+func NewController(mrg manager.Manager, gvk schema.GroupVersionKind, namespace string, runner runner.Runner) {
```

Comment:
I suggest calling this `New` so it can be used as `controller.New()`

Reply:
FWIW, `mrg` seems unintuitive to me even just reading this code. Assuming it's an abbreviation for "manager", `mgr` seems more natural since it doesn't rearrange the letters. Is there some other intent?

Otherwise, I end up having to read `mrg` as "merge", or perhaps "Mr. G". ;)

Reply:
+1